bpf/cpu/cpu.bpf.c

```diff
@@ -182,15 +192,30 @@ typedef struct {
   mapping_t mappings[MAX_MAPPINGS_PER_PROCESS];
 } process_info_t;
 
-// A row in the stack unwinding table for x86_64.
-typedef struct __attribute__((packed)) {
-  u64 pc;
-  u8 cfa_type;
-  u8 rbp_type;
-  s16 cfa_offset;
-  s16 rbp_offset;
-} stack_unwind_row_t;
-_Static_assert(sizeof(stack_unwind_row_t) == 14, "unwind row has the expected size");
+ #if __TARGET_ARCH_arm64
```

Comment:
Given that most fields are shared, perhaps we could do something like:


```c
   // A row in the stack unwinding table.
  typedef struct __attribute__((packed)) {
    u64 PC;
#if __TARGET_ARCH_arm64
    s16 lr_offset;
#endif
    u8 cfa_type;
    u8 rbp_type;
    s16 cfa_offset;
    s16 rbp_offset;
  } stack_unwind_row_t;
```

Reply:
oh wow I didn't know this was even possible! doesn't look great but so cool!